apps/src/code-studio/pd/application_dashboard/detail_view.jsx

```diff
@@ -38,6 +38,12 @@ export default class DetailView extends React.Component {
     });
   }
 
+  updateProps = (newProps) => {
```

Comment:
I would prefer the name to refer to state and not props...maybe `updateData` or `updateStateData`?